app/workflow/wf_service.py

```diff
@@ -0,0 +1,37 @@
+from prefect import Client
+from fastapi import HTTPException
+
+# Replace with Prefect server host:port
+prefect_server = "http://20.25.85.193:4200/graphql"
```

Comment:
externalizing this URI would be great - i don't have anything set up yet so if you'd like: https://www.dynaconf.com is a good way to go about it